apps/src/templates/BaseDialog.jsx

```diff
@@ -75,11 +75,13 @@ var BaseDialog = React.createClass({
       });
     }
 
+    let wrapperClassNames = "";
     let modalClassNames = "modal";
     let modalBodyClassNames = "modal-body";
     let modalBackdropClassNames = "modal-backdrop";
 
     if (this.props.useUpdatedStyles) {
```

Comment:
Originally added this to support the [`AchievementDialog`](https://code-dot-org.github.io/cdo-styleguide/?selectedKind=AchievementDialog&selectedStory=Overview) for gamification.  I'll extract this to a separate component (that doesn't rely on external styles!) in a separate PR.